talent/application-process.md

```diff
@@ -4,27 +4,38 @@ This document describes the source{d} hiring process from the point of view of t
 
 ### Visiting our careers page
 
-- All the open roles can be found at https://sourced.tech/company/#careers where the developer is able to apply to any desired role.
+- All the open roles can be found at https://sourced.tech/company/#careers where the candidate is able to apply to any desired role.
 
-- If there isn't a role that fits but the developer is interested in working with us, they can send an email to talent@sourced.tech with their GitHub account and a small intro about themselves.
+- If there isn't a role that fits but the candidate is interested in working with us, they can send an email to talent@sourced.tech with their CV ( or GitHub account for developers) and a small intro of themselves.
 
 ### Screening
 
-- Less than a week after the submission of the application the developer receives an email with feedback; however there can be delays. In any case, the developer is free to reach out to us at any moment at talent@sourced.tech.
+- Within a week after the submission of the application, the candidate should receive an email with an update. There may occasionally be delays. In any case, the candidate is free to reach out to us at any time via talent@sourced.tech.
 
-- In the case of positive feedback in the application, the developer is asked to move to a quick Skype/Zoom/Phone call with the lead of the potential hiring team.
+- If the application receives positive feedback from our initial screen, the candidate is invited to a brief video call with the Talent Acquisition Manager.
 
-### Skype/Zoom/Phone call
+### Initial interview: 1st stage
 
-- The developer has a quick call with the lead engineer of the team they applied to. This should be a general discussion that can last between 15-30 minutes. Its purpose is to get to know the background and experience of the developer, as well as allowing the developer to get to know source{d} better. It's expected that the developer has already done some basic research about source{d} and has specific questions about the role and/or the company to be expressed during the call. 
+- The candidate has an introductory video call with the Talent Acquisition Manager. This is a general discussion that can last between 15-30 minutes. This is to allow us to get to know the background and experience of the candidate, as well as allowing them to get to know source{d} better. It's expected that the candidate has already done some basic research about source{d} and has some specific questions about the role and/or the company to be expressed during the call. 
 
-- There are notable exceptions to this, such as higher volume junior roles where we directly ask the developer to do the coding challenge before a call.
+- In some cases, notably for junior engineering roles where there are more candidates, we may ask the developer to do a coding challenge before a call.
```

Comment:
```suggestion
- In some cases, notably for junior engineering roles where there are more candidates, the step below: `Coding Challenge: 3rd stage`, can happen at the beginning of the process so we may ask the developer to do the coding challenge before the call.
```

Reply:
sorry David I'm open to suggestions but in this case I think the example you've given is a bit waffling and complicated. Sorry!

Reply:
just tried to explain that "a coding challenge" to be requested "before a call" will replace the one from 3rd step, not being an extra one; but lgtm if you think it's clear enough with the current description ;)

Reply:
Ah ok I see what you mean- perhaps just replace 'a' coding challenge and call with 'the'?